.github/workflows/create-archiver.yml

```diff
@@ -0,0 +1,122 @@
+---
+name: run-archiver
+
+on:
+  workflow_dispatch:
+    inputs:
+      datasets:
+        description: 'Comma-separated list of datasets to archive (e.g., "ferc2","ferc6").'
+        default: '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","epacems","ferc1","ferc2","ferc6","ferc60","ferc714","gridpathratoolkit","mshamines","nrelatb","phmsagas","usgsuspvdb","vcerare"'
```

Comment:
Since this action will create new instances of every archive that is passed in regardless of whether they exist or not already, maybe we should have an empty default, rather than an all datasets default, to avoid accidentally creating dozens of new drafts that hang around being confusing with different DOIs but the same names as pre-existing real archives?